pkg/worker/k8spodtemplating.go

```diff
@@ -112,6 +126,40 @@ func sanitizePodName(name string) string {
 	return name
 }
 
+func getOwnerReferences() []metav1.OwnerReference {
+	var ownerEnabled bool
+	var ownerName string
+	var ownerUID string
+	if err := env.BindMultiple(map[any]string{
+		&ownerEnabled: "WHARF_KUBERNETES_OWNER_ENABLE",
+		&ownerName:    "WHARF_KUBERNETES_OWNER_NAME",
+		&ownerUID:     "WHARF_KUBERNETES_OWNER_UID",
+	}); err != nil {
+		log.Warn().WithError(err).Message("Failed binding WHARF_KUBERNETES_OWNER_XXX environment variables.")
+		ownerEnabled = false
+	}
+
+	log.Debug().
+		WithBool("ownerEnabled", ownerEnabled).
+		WithString("ownerName", ownerName).
+		WithString("ownerUID", ownerUID).
```

Comment:
Same here, could remove the prefix, as it's already stated in the message text.

```suggestion
		WithBool("enabled", ownerEnabled).
		WithString("name", ownerName).
		WithString("uid", ownerUID).
```